src/impl_/pymodule.rs

```diff
@@ -5,9 +5,16 @@ use std::{cell::UnsafeCell, marker::PhantomData};
 #[cfg(all(
     not(any(PyPy, GraalPy)),
     Py_3_9,
-    not(all(windows, Py_LIMITED_API, not(Py_3_10)))
+    not(all(windows, Py_LIMITED_API, not(Py_3_10))),
+    not(target_has_atomic = "64"),
 ))]
 use portable_atomic::{AtomicI64, Ordering};
+#[cfg(all(
+    not(any(PyPy, GraalPy)),
+    Py_3_9,
+    not(all(windows, Py_LIMITED_API, not(Py_3_10))),
+))]
+use std::sync::atomic::{AtomicI64, Ordering};
```

Comment:
Don't we need `target_has_atomic = "64"` here?

Reply:
I would rather ask why do we want to second guess the same logic which `portable_atomic` should do itself, i.e. `portable_atomic::AtomicI64` already should essentially be an alias for `std::sync::atomic::AtomicI64` if the target supports that. So why do we want to make the dependency conditional?

Reply:
`portable_atomic` relies on a `build.rs` which are relatively heavy-weight in terms of compile time, so the goal here is to minimize build time.

Reply:
Agreed, I remember that discussion now and making the build more hermetic is certainly a worthwhile goal.